### PR TITLE
Wa 7 weather on home page

### PR DESCRIPTION
**Pull request by ketryin**

### CODE REVIEW

File: weather-app/src/app/app.component.html [hunk 1]
```diff
@@ -1,2 +1,5 @@
-<div class="main-container"></div>
+<div class="main-container">
+    <app-current-weather></app-current-weather>
```
Comment (Anurov-Prihodko) [inline weather-app/src/app/app.component.html]: i think app-current-weather must position inside a home component

Comment (ketryin) [reply weather-app/src/app/app.component.html]: I will import this component in next pr

Comment (ketryin) [reply weather-app/src/app/app.component.html]: I forgot, I did it in commit [[Wa-7]: addes change in structure.

File: weather-app/src/app/shared/components/current-weather/current-weather.component.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div class="current-weather">
+    <div class="current-weather__info">
+        <span class="icon icon-calendar"></span>
```
[the remark this answers is not in the document]
Comment (ketryin) [reply weather-app/src/app/shared/components/current-weather/current-weather.component.html]: Done

**Pull request merged**